src/components/EditMenu.vue

```diff
@@ -152,7 +152,7 @@
     </div>
     <v-divider class="opacity-20" />
     <div
-      class="flex flex-row justify-start relative items-center bg-[#CBCBCB2A] elevation-5 2xl:h-full xl:h-[45px] h-[35px] overflow-hidden"
+      class="flex flex-row justify-start relative items-center bg-[#CBCBCB2A] elevation-5 2xl:h-[55px] xl:h-[45px] h-[35px] overflow-hidden"
```

Comment:
This is not fixing the full-height problem.
We can just remove this commit as its not necessary to this PR, merge the PR and attack this problem on a later one.

<img width="772" alt="image" src="https://github.com/user-attachments/assets/f3db4df2-36f8-4bc6-b5b2-c3456c4ba3df">

Reply:
> This is not fixing the full-height problem. We can just remove this commit as its not necessary to this PR, merge the PR and attack this problem on a later one.

Removed!